test-suite/rust-runtime-api/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod proto;
+mod service;
+
+#[cfg(test)]
+mod rust_runtime_api_tests {
+    use assert_matches::assert_matches;
+    use exonum_rust_runtime::{api::Error as ApiError, ProtoSourceFile, ProtoSourcesQuery};
+    use pretty_assertions::assert_eq;
+
+    use std::collections::HashSet;
+
+    use crate::service::TestRuntimeApiService;
+    use exonum_testkit::{ApiKind, TestKit, TestKitApi, TestKitBuilder};
+
+    fn testkit_with_rust_service() -> (TestKit, TestKitApi) {
```

Comment:
This function should probably be moved outside the test code and made public so that the compiler doesn't complain about dead code.